tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/connectors/sql/SQLActionsTest.java

```diff
@@ -136,7 +136,15 @@ public void testGeneratedKeyOnInsert() {
     public void testGeneratedKeyOnInsertEmptyResults() {
         BValue[] returns = BRunUtil.invoke(result, "testGeneratedKeyOnInsertEmptyResults", connectionArgs);
         BInteger retValue = (BInteger) returns[0];
-        Assert.assertEquals(retValue.intValue(), 0);
+
+        int columnCount;
+        if (dbType == POSTGRES) {
+            // Postgres returns all columns when there is no returning clause
+            columnCount = 5;
+        } else {
+            columnCount = 0;
+        }
```

Comment:
We can use ?: expression to simplify this:
`int columnCount = (dbType == POSTGRES) ? 5 : 0;`